src/Sylius/Component/Attribute/AttributeType/ChoiceAttributeType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Attribute\AttributeType;
+
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * @author Saša Stamenković <umpirsky@gmail.com>
+ */
+class ChoiceAttributeType implements AttributeTypeInterface
+{
+    const TYPE = 'choice';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStorageType()
+    {
+        return AttributeValueInterface::STORAGE_TEXT;
```

Comment:
I see 2 options:

* Adding JSON storage which will be added anyway and using it here.
* Adding JSON storage but introducing 2 attribute types, single + multiple select, using text + json storages respectively for these two.